src/ConfigurationsProvider.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactNode, createContext, useContext, useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
+
+import { SettingsContext } from './SettingsProvider';
+import { defaultAppDesignSystemConfig, defaultSueAppConfig } from './config/configurations';
+import { useStaticContent } from './hooks';
+import { QUERY_TYPES, getQuery } from './queries';
+import { storageHelper } from './helpers';
+
+const defaultConfiguration = {
+  appDesignSystem: defaultAppDesignSystemConfig,
+  sueConfig: defaultSueAppConfig
+};
+
+export const ConfigurationsContext = createContext(defaultConfiguration);
+
+export const ConfigurationsProvider = ({ children }: { children?: ReactNode }) => {
+  const { globalSettings } = useContext(SettingsContext);
+  const { settings, appDesignSystem = {} } = globalSettings;
+  const { sue = {} } = settings || {};
+
+  const [configurations, setConfigurations] = useState(defaultConfiguration);
+
+  const { data: sueConfigData } = useQuery(
+    [QUERY_TYPES.SUE.CONFIGURATIONS],
+    () => getQuery(QUERY_TYPES.SUE.CONFIGURATIONS)(),
+    { enabled: !!sue }
```

Comment:
check this:

<img width="71" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/db208e9b-4fc0-4973-9c16-559eed302d0b">

i think you need to check more deeply, if `sue` data is given

Reply:
How about a control like that?
`!!Object.keys(sue).length;`